docs/bcfusion/validators/staking.md

```diff
@@ -0,0 +1,133 @@
+# Staking
+
+BNB smart chain (BSC) is a Proof-of-Staked-Authority (PoSA) blockchain, which means that staking is one of the most
+important parts of the system. [BEP-294](https://github.com/bnb-chain/BEPs/pull/294) introduces the new native staking
+mechanism after BNB chain fusion, which has several improvements:
+
+- Users are able to participate in staking on the BSC using BNB in BEP20 format.
+- Staking credit, as the proof of BNB staked to a specified validator, cannot be transferred. Different validators issue
+  different staking credits.
+- The rewards for staking can be claimed more quickly.
+
+In this section, we will explain the basic concepts and operations of staking on the BSC.
+
+## Validator Set
+
+The validator set is the group of nodes that are responsible for validating transactions and producing blocks on the
+BSC. The validator set is determined by the amount of staking credit each validator has, which reflects the amount of
+BNB staked by the validator and its delegators. The top 45 validators with the most staking credit are selected as the
```

Comment:
`45` is a temp value, will grow to 100 and more in the future

Reply:
fixed